examples/config/config.properties

```diff
@@ -0,0 +1,267 @@
+[[periods]]
+name = "ea"
+length = 3
+capacity_factor = 3
+
+[[periods]]
+name = "am"
+length = 4
+capacity_factor = 3.65
+
+[[periods]]
+name = "md"
+length = 5
+capacity_factor = 5
+
+[[periods]]
+name = "pm"
+length = 4
+capacity_factor = 3.65
+
+[[periods]]
+name = "ev"
+length = 8
+capacity_factor = 8
+
+[run]
+air_passenger = true
+non_motorized_skim = true
+highway_skim = true
+transit_skim = true
+global_iterations = 3
+
+[emme]
+num_processors = "max-2"
+
+[emme.scenario_ids]
+ea = 1
+am = 2
+md = 3
+pm = 4
+ev = 5
+
+[emme.highway_maz]
+operating_cost = 18.93
+value_of_time = 17.23
+
+[emme.highway]
+relative_gap = 0.0005
+max_iterations = 30
+tollbooth_start_index = 11
+[[emme.highway.classes]]
+name = "da"
+description = "drive alone"
+emme_mode = "d"
+excluded_links = [ "is_toll_da", "is_sr2",]
+value_of_time = 18.93
+operating_cost = 17.23
+toll = "@bridgetoll_da"
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_da",]
+[[emme.highway.classes.demand]]
+file = "household"
+name = "SOV_GP_{period}"
+
+[[emme.highway.classes.demand]]
+file = "air_passenger"
+name = "DA"
+
+[[emme.highway.classes.demand]]
+file = "internal_external"
+name = "DA"
+
+
+[[emme.highway.classes]]
+name = "sr2"
+description = "shared ride 2"
+emme_mode = "e"
+excluded_links = [ "is_toll_sr2", "is_sr3",]
+value_of_time = 18.93
+operating_cost = 17.23
+toll = "@bridgetoll_sr2"
+skims = [ "time", "dist", "freeflowtime", "bridgetoll_sr2", "hovdist",]
+[[emme.highway.classes.demand]]
+file = "household"
+name = "SR2_GP_{period}"
+factor = 0.5714285714285714
+
+[[emme.highway.classes.demand]]
+file = "household"
+name = "SR2_HOV_{period}"
+factor = 0.5714285714285714
+
+[[emme.highway.classes.demand]]
+file = "air_passenger"
+name = "SR2"
+
+[[emme.highway.classes.demand]]
+file = "internal_external"
+name = "SR2"
+
+
+[[emme.highway.classes]]
+name = "sr3"
+description = "shared ride 3+"
+emme_mode = "f"
+excluded_links = [ "is_toll_sr3",]
```

Comment:
For all of these excluded link dummy variables, I think these would be made in Lasso - but want to confirm with @lmz  - and will want to add as issue to lasso to do.

@inrokevin can you add to documentation a list of all these variables that you are expecting in the network?